src/Sounder/Sounder.tsx

```diff
@@ -1,68 +1,72 @@
+  )
 
-    const handleLength = (longness: boolean): string => {
-        if (!longness) {
-            return "50n"
-        }
-        else {
-            return "8n"
-        }
-    }
+  const handleSound = (): void => {
+    const randomInteger = Math.floor(Math.random() * availableNotes.length)
+    const randomNote: Note.Note = availableNotes[randomInteger]
+    //handleNote()
+    synth.triggerAttackRelease(Note.getNote(randomNote), handleLength(length))
+    startTransition(() => {
+      setCurrentNote(randomNote)
+    })
+  }
 
-    const handleNote =() : void =>{
-        const random = Math.floor(Math.random() * availableNotes.length);
-        setNote(availableNotes[random])
+  const handleLength = (longness: boolean): string => {
+    if (!longness) {
+      return "50n"
+    } else {
+      return "8n"
     }
+  }
 
-    return (
-        <div className='flex-column align-center'>
-            <div className='flex align-center '>
-                <header className='text-purple-400 px-3 flex-auto'>long notes -{'>'}</header>
-                <div id='length-switch flex-auto' className='p-0'>
-                    <Switch
-                        checked={long}
-                        onChange={setLength}
-                        className={`${long ? 'bg-purple-900' : 'bg-purple-700'} relative inline-flex h-[38px] w-[74px] shrink-0 cursor-pointer rounded-full border-2 border-gray-400 transition-colors duration-200 ease-in-out focus:outline-none focus-visible:ring-2  focus-visible:ring-white focus-visible:ring-opacity-0 flex-auto`}
-                    >
-                        <span
-                            aria-hidden="true"
-                            className={`${long ? 'translate-x-9' : 'translate-x-0'} pointer-events-none inline-block h-[34px] w-[34px] transform rounded-full bg-white shadow-lg ring-0 transition duration-200 ease-in-out`}
-                        />
-                    </Switch>
-                </div>
-            </div>
-            <div>
-                <button onClick={handleSound}>
-                    <VolumeUpIcon className="App-logo animate-pulse text-icon-color" />
-                </button>
-                <header className="text-yellow-500">Note: {currentNote}</header>
-            </div>
+  return (
+    <div className="flex-column align-center">
+      <div className="flex align-center ">
+        <header className="text-purple-400 px-3 flex-auto">
+          long notes -{">"}
+        </header>
+        <div id="length-switch flex-auto" className="p-0">
+          <Switch
+            checked={length}
+            onChange={setLength}
+            className={`${
+                length ? "bg-purple-900" : "bg-purple-700"
+            } relative inline-flex h-[38px] w-[74px] shrink-0 cursor-pointer rounded-full border-2 border-gray-400 transition-colors duration-200 ease-in-out focus:outline-none focus-visible:ring-2  focus-visible:ring-white focus-visible:ring-opacity-0 flex-auto`}
```

Comment:
this is getting a little hard to read. you might consider pulling out some constants:

```typescript
const baseStyle = "relative inline-flex ..."
const switchStyle = length ? `bg-purple-900 ${baseStyle}` : `bg-purple-700 ${baseStyle}`

...

<Switch className={switchStyle} ... />
```